ansible/roles/base/tasks/main.yml

```diff
@@ -1,14 +1,12 @@
-
```

Comment:
Personally, I use PyCharm from JetBrains (relatively inexpensive). Some folks also like Atom (Free), but that may require some customization.